### PR TITLE
[#696]upgrade spring boot and spring could version

### PR DESCRIPTION
**Pull request by zhaozw**

### CODE REVIEW

**Review by Shoothzj — APPROVED** [review]
LGTM

**Comment by Shoothzj** [comment]
@zhaozw does `spring-cloud` 3.1.2 still use spring-boot 2.6.7?

**Comment by zhaozw** [comment]
> @zhaozw does `spring-cloud` 3.1.2 still use spring-boot 2.6.7?

I think so,
spring-cloud-dependencies pom:
<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <parent>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-dependencies-parent</artifactId>
    <version>3.1.2</version>
    <relativePath></relativePath>
  </parent>
  <groupId>org.springframework.cloud</groupId>
  <artifactId>spring-cloud-dependencies</artifactId>
  <version>2021.0.2</version>
  <packaging>pom</packaging>
  <name>spring-cloud-dependencies</name>
  <description>Spring Cloud Dependencies</description>



The spring-cloud-dependencies-parent version is 3.1.2,and the spring-cloud-build 3.1.2 pom:


<?xml version="1.0" encoding="UTF-8"?>
<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
		 xmlns="http://maven.apache.org/POM/4.0.0"
		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>
	<groupId>org.springframework.cloud</groupId>
	<artifactId>spring-cloud-build</artifactId>
	<version>3.1.2</version>
	<packaging>pom</packaging>
	<name>Spring Cloud Parent</name>
	<description>Spring Cloud parent pom, managing plugins and dependencies for Spring
		Cloud projects
	</description>
	<modules>
		<module>docs</module>
		<module>spring-cloud-build-dependencies</module>
		<module>spring-cloud-dependencies-parent</module>
		<module>spring-cloud-build-tools</module>
	</modules>
	<url>https://spring.io/spring-cloud</url>
	<properties>
		<java.version>1.8</java.version>
		
		<resource.delimiter>@</resource.delimiter>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
		<maven.compiler.source>${java.version}</maven.compiler.source>
		<maven.compiler.target>${java.version}</maven.compiler.target>
		<main.basedir>${basedir}</main.basedir>
		<docs.main>${project.artifactId}</docs.main>
		<spring-boot.version>2.6.7</spring-boot.version>
		<spring-cloud-build.version>3.1.2</spring-cloud-build.version>

File: spring-cloud-starter-huawei/spring-cloud-starter-huawei-swagger/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>spring-cloud-starter-huawei</artifactId>
     <groupId>com.huaweicloud</groupId>
-    <version>1.10.0-2020.0.x-SNAPSHOT</version>
+    <version>1.10.0-2021.0.x-SNAPSHOT</version>
```
Comment (liubao68) [inline spring-cloud-starter-huawei/spring-cloud-starter-huawei-swagger/pom.xml:24]: 建议先不更新版本后，等版本功能基本可以发布的时候，再更新。方便后面rebase master的修改。

**Comment by liubao68** [comment]
> @zhaozw does `spring-cloud` 3.1.2 still use spring-boot 2.6.7?

see https://spring.io/projects/spring-cloud

spring cloud latest version still use spring boot 2.6.x

File: spring-cloud-starter-huawei/spring-cloud-starter-huawei-swagger/pom.xml (hunk 1 above)
Comment (Shoothzj) [reply spring-cloud-starter-huawei/spring-cloud-starter-huawei-swagger/pom.xml:24]: agree

Comment (zhaozw) [reply spring-cloud-starter-huawei/spring-cloud-starter-huawei-swagger/pom.xml]: done

**Review by liubao68 — APPROVED** [review]

**Pull request merged**